go.mod

```diff
@@ -5,6 +5,5 @@ go 1.12
 require (
 	github.com/casbin/casbin/v2 v2.0.0
 	github.com/globalsign/mgo v0.0.0-20181015135952-eeefdecb41b8
```

Comment:
github.com/globalsign/mgo is still there.

Reply:
Sorry,I forgot this